lib/index.js

```diff
@@ -75,10 +78,35 @@ Mixpanel.prototype.identify = function(identify, fn){
     data: b64encode(payload)
```

Comment:
Move `query` inside the `batch.push` call; data should be as local as possible, and we only use it in there